t/compat.t

```diff
@@ -239,7 +239,7 @@ ok $mb, "Module::Build->new_from_context";
     );
 
     require ExtUtils::Install;
-    skip "Needs ExtUtils::Install 1.32 or later", 2 * @cases
+    skip "Needs ExtUtils::Install 1.32 or later", int(2 * @cases)
```

Comment:
> On May 10, 2016, at 6:47 PM, Karen Etheridge <notifications@github.com> wrote:
> 
> In t/compat.t:
> 
> > @@ -239,7 +239,7 @@ ok $mb, "Module::Build->new_from_context";
> >      );
> >  
> >      require ExtUtils::Install;
> > -    skip "Needs ExtUtils::Install 1.32 or later", 2 * @cases
> > +    skip "Needs ExtUtils::Install 1.32 or later", int(2 * @cases)
> 
> How can this change possibly make a difference? The number of elements in a list cannot not be an integer.

The number of elements in an array needs to be an integer, and cannot be a string nor number. 
2*@cases return :Number, but needs to be :Int

That’s why cperl complains about that at compile-time. It doesn’t add this cast automatically.

Reini Urban
rurban@cpan.org